### PR TITLE
Re-attaching hand objects to the camera rig.

### LINKED ISSUE #220: regression: controllers in VR do not work on master
this appears to be a regression from PR #213:

image

<hr>

**reproducible platform:** HTC Vive; Windows 10; Firefox Nightly, Chrome Canary

### PR DESCRIPTION
**Pull request by delapuente**
Fix #220

### CODE REVIEW

File: Assets/WebVR/Scenes/WebVR.unity [hunk 1]
```diff
@@ -1601,6 +1601,16 @@ Prefab:
       propertyPath: m_RootOrder
       value: 0
       objectReference: {fileID: 0}
+    - target: {fileID: 114951743782495552, guid: 78df02845d4b54181b4ca9188aba0671,
```
Comment (cvan) [inline Assets/WebVR/Scenes/WebVR.unity]: ah, this happened right here!

https://github.com/mozilla/unity-webvr-export/pull/213/files#diff-87eb99a2d1e7b48246e4f7c07d3b7cf2L1604

I don't know how I didn't catch that in my final round of testing. thanks for fixing!

File: Assets/WebVR/Scenes/WebVR.unity [hunk 2]
```diff
@@ -1601,6 +1601,16 @@ Prefab:
       propertyPath: m_RootOrder
       value: 0
```
Comment (cvan) [inline Assets/WebVR/Scenes/WebVR.unity]: you changed the [`value`s here](https://github.com/mozilla/unity-webvr-export/pull/213/files#diff-87eb99a2d1e7b48246e4f7c07d3b7cf2L791) - you intend to revert those too?

**Comment by cvan** [comment]
FYI: both @caseyyee and I have experienced the same issue: the position of the hands appear offset + backwards

File: Assets/WebVR/Scenes/WebVR.unity (hunk 2 above)
Comment (delapuente) [reply Assets/WebVR/Scenes/WebVR.unity]: Per consistency, we always use the _left first, then right_ so I thought it was fine to make the game objects for hands to be consistent.

**Pull request merged**